v6000.4.0f1 error

I updated my project to v6000.4.0f1 and I'm getting this error on the console and the blur material is not working:

"Packages\com.unify.unified-universal-blur\Runtime\UniversalBlurPass.cs(60,30): error CS0115: 'UniversalBlurPass.Execute(ScriptableRenderContext, ref RenderingData)': no suitable method found to override"

Can someone help?

--- a/Runtime/UniversalBlurPass.cs
+++ b/Runtime/UniversalBlurPass.cs
@@ -57,6 +57,7 @@ namespace Unified.UniversalBlur.Runtime
                 autoGenerateMips = _blurConfig.EnableMipMaps
             };
 
+#if !UNITY_6000_4_OR_NEWER
         public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
         {
             var cmd = CommandBufferPool.Get();
@@ -91,7 +92,8 @@ namespace Unified.UniversalBlur.Runtime
             context.ExecuteCommandBuffer(cmd);
             CommandBufferPool.Release(cmd);
         }
-
+#endif
+        
 #if UNITY_6000_0_OR_NEWER
         public override void RecordRenderGraph(RenderGraph renderGraph, ContextContainer frameData)
         {

